src/python/pants/backend/python/lint/bandit/rules_integration_test.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from typing import List, Optional
+
+from pants.backend.python.lint.bandit.rules import BanditTarget
+from pants.backend.python.lint.bandit.rules import rules as bandit_rules
+from pants.backend.python.lint.bandit.subsystem import Bandit
+from pants.backend.python.rules import download_pex_bin, pex
+from pants.backend.python.rules.pex import CreatePex
+from pants.backend.python.subsystems import python_native_code, subprocess_environment
+from pants.backend.python.subsystems.python_native_code import PythonNativeCode
+from pants.backend.python.subsystems.python_setup import PythonSetup
+from pants.backend.python.subsystems.subprocess_environment import SubprocessEnvironment
+from pants.backend.python.targets.python_library import PythonLibrary
+from pants.build_graph.address import Address
+from pants.build_graph.build_file_aliases import BuildFileAliases
+from pants.engine.fs import FileContent, InputFilesContent, Snapshot
+from pants.engine.legacy.structs import PythonTargetAdaptor
+from pants.engine.rules import RootRule
+from pants.engine.selectors import Params
+from pants.rules.core.lint import LintResult
+from pants.source.wrapped_globs import EagerFilesetWithSpec
+from pants.testutil.option.util import create_options_bootstrapper
+from pants.testutil.subsystem.util import init_subsystems
+from pants.testutil.test_base import TestBase
+
+
+class BanditIntegrationTest(TestBase):
+  good_source = FileContent(path="test/good.py", content=b"hashlib.sha256()\n")
+  bad_source = FileContent(path="test/bad.py", content=b"hashlib.md5()\n")
+  # MD5 is a insecure hashing function
+
+  @classmethod
+  def alias_groups(cls) -> BuildFileAliases:
+    return BuildFileAliases(targets={'python_library': PythonLibrary})
+
+  @classmethod
+  def rules(cls):
+    return (
+      *super().rules(),
+      *bandit_rules(),
+      *download_pex_bin.rules(),
+      *pex.rules(),
+      *python_native_code.rules(),
+      *subprocess_environment.rules(),
+      RootRule(CreatePex),
+      RootRule(Bandit),
+      RootRule(BanditTarget),
+      RootRule(PythonSetup),
+      RootRule(PythonNativeCode),
+      RootRule(SubprocessEnvironment),
+    )
+
+  def setUp(self):
+    super().setUp()
+    init_subsystems([Bandit, PythonSetup, PythonNativeCode, SubprocessEnvironment])
```

Comment:
These can be removed now that we use `OptionsBootstrapper`, which will already initialize those subsystems to use the same defaults that are used in production.

Reply:
Good catch, thx!